_posts/2020-06-20-Basic-Querying.md

```diff
@@ -0,0 +1,302 @@
+
+This is [the pedigree of Dál nAraide](https://celt.ucc.ie//published/G800011F/text063.html) that featured in [a previous blogpost](https://cyocum.github.io/2020/06/14/Emendations-to-Source-Editions-on-IrishGen.html). The text **in bold** is a marginal insertion, which both the scribe and the editors of **[L](https://www.vanhamel.nl/codecs/Best,_et_al._1954-1983)** place in [a Cenél Lóegaire pedigree](http://www.ucc.ie/celt/published/G800011F/text058.html) but which we feel makes far more sense if read here.
+
+This passage is an origin legend and etymological narrative for the Ulster kin-group of Dál nAraide, whose kingdom was located to the east of that of Cíannachta Glinne Geimen, stretching from Mag nEilne, on the north coast between the [River Bush](https://www.logainm.ie/116127.aspx ) and the [River Bann](https://www.logainm.ie/1165494.aspx), down to [Lough Neagh](https://www.logainm.ie/1166400.aspx). They were a major power in Ulster in middle of the first millennium. Genealogically, according to this text, they are rooted in the north, descending from the Ulster Cycle hero, Conall Cernach. However, like the Cíannachta, their story is also linked via etymology in to that of Cormac mac Airt (= Cormac, grandson of Conn) and the legendary kingship of Ireland, which in turn connects them to far-off Munster. The compilation of genealogical collections as we have them was, after all, part of the developing national pseudo-history of Ireland, in which the idea of an island-wide kingship was key.
+
+Araide Bibre, our satirist, is the linchpin of the etymological, 'national' side of this narrative. The significance of him being a satirist is unclear. It does not mean that he was ignoble, however. Satire was the inverse of praise poetry in early Ireland but still took place in the context of elite politics, supposedly within a regulatory framework. For a detailed recent study, see Roisín McLoughlin's _[Early Irish Satire](http://www.worldcat.org/oclc/495267258)_. A satirist, whatever the term's modern connotations, would have been very much part of the establishment, even though illegitimate use of satire appears to have been a concern among jurists.
```

Comment:
Could you find a reference for `appears to have been a concern among jurists`?  Early Irish Satire covers this pp. 3-8 so you could reference that.

Reply:
Thanks! I read a review of _Early Irish Satire_ and was basing it on that.

Reply:
But a proper page reference is much better.